contracts/src/v0.8/ccip/test/helpers/BurnMintERC677Helper.sol

```diff
@@ -2,17 +2,16 @@
 pragma solidity 0.8.24;
 
 import {BurnMintERC677} from "../../../shared/token/ERC677/BurnMintERC677.sol";
-import {IGetCCIPAdmin} from "../../interfaces/IGetCCIPAdmin.sol";
 
-contract BurnMintERC677Helper is BurnMintERC677, IGetCCIPAdmin {
+contract BurnMintERC677Helper is BurnMintERC677 {
```

Comment:
Why was this removed?

Reply:
I probably removed it somewhere in the development process when it was causing inheritance issues and forgot to put it back in at some point. This is a helper contract does it matter?

Reply:
Why would you remove it if it doesn't matter?

Reply:
I mean I had the opposite belief, that if it didn't matter then remove it for readability but thats fine i'll re-add it. I'm pretty sure interfaces don't affect the bytecode anyways